src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/AzureFiles/ProtectionCheckTests.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Test.ScenarioTests
 {
     public partial class ProtectionCheckTests : RMTestBase
     {
-        [Fact]
+        [Fact(Skip = "To un-skip in upcoming release")]
```

Comment:
Let me just leave a comment as a reminder to unskip this case

Reply:
Ack.